src/objective.jl

```diff
@@ -131,7 +131,7 @@ function MOI.get(m::LinQuadOptimizer, ::MOI.ObjectiveFunction{Quad})
         for j in nzrange(Q, i)
             row = rows[j]
             val = vals[j]
-            push!(quadratic_terms, MOI.ScalarQuadraticTerm{Float64}(0.5*val, m.variable_references[row], m.variable_references[i]))
+            push!(quadratic_terms, MOI.ScalarQuadraticTerm{Float64}(val, m.variable_references[row], m.variable_references[i]))
```

Comment:
We need tests to catch this

Reply:
Yes! I am not even sure that is the right way